firestore/tests/unit/test_collection.py

```diff
@@ -427,12 +427,12 @@ def _call_fut():
     def test_it(self, mock_rand_choice):
         from google.cloud.firestore_v1beta1.collection import _AUTO_ID_CHARS
 
-        mock_result = '0123456789abcdefghij'
+        mock_result = '23456789abcdefghjkmnp'
         mock_rand_choice.side_effect = list(mock_result)
         result = self._call_fut()
         self.assertEqual(result, mock_result)
 
-        mock_calls = [mock.call(_AUTO_ID_CHARS)] * 20
+        mock_calls = [mock.call(_AUTO_ID_CHARS)] * 21
```

Comment:
It should be 20 characters in length.

Reply:
@schmidt-sebastian 21 is correct (the change in this PR is dropping set of possible characters, so we add an extra character to keep the entropy up)

Reply:
The decreased level of entropy seems more acceptable than having auto-ids that differ in length between implementations. If you are worried about the entropy, I would much rather suggest dropping this PR :)

Reply:
That works for me. Dropping it.